src/components/UpDownButton.tsx

```diff
@@ -17,10 +17,10 @@ const UpDownButton = () => {
     const viewportHeight = document.documentElement.clientHeight;
 
     const atBottom =
-      pathname === '/attendance-overview' ? scrolled >= 300 : scrolled >= 720;
+      pathname === '/attendance-overview' ? scrolled >= 294 : scrolled >= 700;
```

Comment:
@shreyas1434shinde have u tested this on multiple devices?

Reply:
@itsvick yes